src/commands/owner/eval.js

```diff
@@ -0,0 +1,65 @@
+const { Command } = require("@src/structures");
+const { MessageEmbed } = require("discord.js");
+
+module.exports = class Eval extends Command {
+  constructor(client) {
+    super(client, {
+      name: "eval",
+      description: "evaluates something",
+      command: {
+        enabled: true,
+        usage: "1+1",
+        minArgsCount: 1,
+        category: "OWNER",
+        botOwnerOnly: true,
+        botPermissions: ['EMBED_LINKS']
+      },
+      slashCommand: {
+        enabled: true,
+        options: [
+          {
+            name: "input",
+            description: "content to eval",
+            type: "STRING",
+            required: true,
+          },
+        ],
+      },
+    });
+  }
+
+  /**
+   * @param {Message} message
+   * @param {string[]} args
+   */
+  async messageRun(message, args) {
+
+        const input = args.join(' ');
+        if (!input) return message.reply('Please provide code to eval');
+        if(!input.toLowerCase().includes('token')) {
+    
+          const embed = new MessageEmbed();
+    
+          try {
+            let output = eval(input);
```

Comment:
Where is the eval () function defined?

Reply:
![](https://cdn.discordapp.com/attachments/881752599232118855/882485304936771604/unknown.png)